Tools/YamlCreate.ps1

```diff
@@ -1,262 +1,261 @@
-[Console]::OutputEncoding = [System.Text.Encoding]::UTF8
-
-# The intent of this file is to help you generate a YAML file for publishing
-# to the Windows Package Manager repository.
+#Requires -Version 5
+
+<#
+.SYNOPSIS
+    Winget Manifest creation helper script
+.DESCRIPTION
+    The intent of this file is to help you generate a manifest for publishing
+    to the Windows Package Manager repository. 
+    
+    It'll attempt to download an installer from the user-provided URL to calculate
+    a checksum. That checksum and the rest of the input data will be compiled in a 
+    .YAML file.
+.EXAMPLE
+    PS C:\Projects\winget-pkgs> Get-Help .\Tools\YamlCreate.ps1 -Full
+    Show this script's help
+.EXAMPLE
+    PS C:\Projects\winget-pkgs> .\Tools\YamlCreate.ps1
+    Run the script to create a manifest file
+.NOTES
+    Please file an issue if you run into errors with this script:
+    https://github.com/microsoft/winget-pkgs/issues/
+.LINK
+    https://github.com/microsoft/winget-pkgs/blob/master/Tools/YamlCreate.ps1
+#>
 
-# define variables
+[Console]::OutputEncoding = [System.Text.Encoding]::UTF8
 $OFS = "`r`n"  #linebreak
```

Comment:
>"`r`n"  #linebreak [](start = 7, length = 18)

Would [Environment]::NewLine be more portable?

Reply:
Good idea, that's definitely more portable. Updated the script 👍